PhoneBook/src/main/java/humphrey/jeriah/PhoneBook.java

```diff
@@ -0,0 +1,52 @@
+package humphrey.jeriah;
+import java.util.*;
+
+/**
+ * Created by jeriahhumphrey on 2/7/17.
+ */
+public class PhoneBook {
+
+     TreeMap <String, String> phoneBook = new TreeMap();
+
+
+     public String lookUp(String key){
+         return phoneBook.get(key);
+     }
+
+     public void addEntry (String name, String number){
+         phoneBook.put(name, number);
+
+     }
+
+     public TreeMap getMap(){
+         return phoneBook;
+     }
+
+     public String listNames() {
+                  String list="";
+         for (Map.Entry<String, String> entries : phoneBook.entrySet()
+                 ) {
+              list += "Name: " + entries.getKey() + ", ";
+
+         }
+         System.out.println(list);
```

Comment:
Rather than printing the list, just returning it should be enough. Sometimes you might want to get a copy of the list without printing it out. Same applies for `listEntries()`